docs/howto/windows.md

```diff
@@ -0,0 +1,34 @@
+Small guide on how to use the terminal using WSL (Windows Subsystem for Linux) with the windows file system, and using it to perform any commands while running a react native android app in Android emulator on Windows.
+
+### Install WSL
+Install your preferred distribution of Linux available in the Windows store (You must have Windows build 1607 or later to be able to do this). I used Ubuntu for this guide.
+
+### Accessing the filesystem
+You can access your Windows local drives by navigating to `/mnt/<drive-letter>`
+
+### Running npm commands
+First install node in your subsystem. For example, to install node 8.x in Ubuntu, run:
+
+`curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -`
```

Comment:
If you write these like this, they will format better:
```
one line
another line
```

Reply:
Fixed!